### PR TITLE
fix wrong preprocess text

### PR DESCRIPTION
**Pull request by trileeeee**

### CODE REVIEW

File: vi_cleaner/abbreviation_vi.py [hunk 1]
```diff
@@ -15,26 +15,18 @@
...
-    key = key.replace(".", "\.").lower()
-    return _abbreviations_vi[key]
-
-
 def normalize_abbreviations_vi(text):
     text = normalize_speacial_symbol_vi(text)
     text = re.sub(_url_re, _expand_urls_vi, text)
-    text = re.sub(_abbreviations_combine_re, _expand_abbreviations_vi, text, flags=re.IGNORECASE)
+    for k, v in _abbreviations_vi.items():
+        text = re.sub(r"\b" + k + r"\b", v, text, flags=re.IGNORECASE)
```
Comment (trong-codelink) [inline vi_cleaner/abbreviation_vi.py]: Maybe we can somehow don't use for loop

Comment (trileeeee) [reply vi_cleaner/abbreviation_vi.py]: Hmm It was not in the past but then the error persists i.e: kounde -> khôngunde

Comment (trileeeee) [reply vi_cleaner/abbreviation_vi.py]: I tried to put the r"\b" between the _abbreviations_vi_list but to no avail, it got processed like this ("\bko\b"|"\bco\b"|")

Comment (trong-codelink) [reply vi_cleaner/abbreviation_vi.py]: Maybe you can try adding `_vietnamese-re` (which is in file symbol_vi.py) to verify the abbr is not in the middle of another word.

Comment (trileeeee) [reply vi_cleaner/abbreviation_vi.py]: done em

Comment (trong-codelink) [reply vi_cleaner/abbreviation_vi.py]: Don't know if it works.

Comment (trileeeee) [reply vi_cleaner/abbreviation_vi.py]: Hi em, I think this is good enough ?
image

Comment (trong-codelink) [reply vi_cleaner/abbreviation_vi.py]: Cool, LGTM!

**Review by trong-codelink — APPROVED** [review]

**Pull request merged**